README

```diff
@@ -334,6 +334,16 @@ in test failure descriptions. Very useful in combination with MonkeyPatch.
   ...     pass
   >>> fixture.cleanUp()
 
+Deprecations
+++++++++++++
+
+Prevent code under test from calling deprecated functions.
+
```

Comment:
imported warnings and now local tox -e py27 works. Also switched to using context for the setUp/cleanUp.